internal/collector/pgbackrest.go

```diff
@@ -107,7 +105,7 @@ func NewConfigForPgBackrestRepoHostPod(
 			Processors: []ComponentID{
 				"resource/pgbackrest",
 				"transform/pgbackrest_logs",
-				SubSecondBatchProcessor,
```

Comment:
Are we going to use this Processor? Seems like we could ditch it (I'm 10 files in)

Reply:
It's still used in some metrics pipelines. 🤔 Worth a review later.